image-mariadb/etc/firstrun/mariadb.sh

```diff
@@ -97,6 +97,8 @@ else
     echo "Setting database file permissions"
     chown -R abc:abc /config/databases
     chmod -R 755 /config/databases
+	echo "Removing mysql-server logrotate directive"
+	rm /etc/logrotate.d/mysql-server
```

Comment:
Please fix spacing to match.

Reply:
Fixed. (Notepad++ was auto indenting with tabs instead of spaces, sorry.